drivers/espi/espi_handlers.c

```diff
@@ -55,7 +55,7 @@ static inline int z_vrfy_espi_write_lpc_request(const struct device *dev,
 	Z_OOPS(Z_SYSCALL_DRIVER_ESPI(dev, write_lpc_request));
 	Z_OOPS(z_user_from_copy(&data_copy, data, sizeof(*data)));
 
-	return z_impl_espi_write_lpc_request(dev, op, data_copy);
+	return z_impl_espi_write_lpc_request(dev, op, &data_copy);
```

Comment:
This looks correct as is now, no need to changed since recently fixed by @ceolin

Reply:
Ok. I will get rid of this line.

Reply:
Done.

Reply:
No, it still doesn't work and block this PR. Please refer https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/espi/espi_handlers.c#L58. Might I ask which PR fix it? If not, I will add it back in this CL.